config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1822,31 +1826,6 @@ def generate_versions():
         )
     return results
 
-######################
-# kops-pipeline.yaml #
-######################
-def generate_pipeline():
```

Comment:
we still need these because all of the other periodic jobs including grid jobs pull from the `latest-ci-updown- green.txt` version markers that the pipeline jobs publish.

https://github.com/kubernetes/test-infra/blob/5ff771e45c26d8168ad020ec659fe8ece878e638/config/jobs/kubernetes/kops/kops-periodics-grid.yaml#L99